From eb0f983459593a3045a7a24c154b66de47894b00 Mon Sep 17 00:00:00 2001
From: Florian Fainelli <f.fainelli@gmail.com>
Date: Sat, 18 Jun 2016 19:28:53 -0700
Subject: [PATCH] net: dsa: b53: SCU Hack change CPU port

On the IMS SCU board, the CPU port has been wired at Port 4, instead of
Port 5 or 8.

Signed-off-by: Florian Fainelli <f.fainelli@gmail.com>
---
 drivers/net/dsa/b53/b53_common.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/net/dsa/b53/b53_common.c b/drivers/net/dsa/b53/b53_common.c
index 274f3679f33d7..dfc5b1309f63e 100644
--- a/drivers/net/dsa/b53/b53_common.c
+++ b/drivers/net/dsa/b53/b53_common.c
@@ -1555,7 +1555,11 @@ static const struct b53_chip_data b53_switch_chips[] = {
 		.vlans = 4096,
 		.enabled_ports = 0x1f,
 		.arl_entries = 4,
+#ifdef CONFIG_SCU
+		.cpu_port = 4,
+#else
 		.cpu_port = B53_CPU_PORT,
+#endif
 		.vta_regs = B53_VTA_REGS,
 		.duplex_reg = B53_DUPLEX_STAT_GE,
 		.jumbo_pm_reg = B53_JUMBO_PORT_MASK,
